runtime/common/src/weights.rs

```diff
@@ -123,6 +130,255 @@ pub fn check_instructions_weights<T: pallet_gear::Config>(
     check_instruction_weight(weights.i32rotr, expected.i32rotr);
 }
 
+/// Check that the weights of syscalls are within the expected range
+pub fn check_syscall_weights<T: pallet_gear::Config>(
+    weights: SyscallWeights<T>,
+    expected: SyscallWeights<T>,
+) {
+    check_syscall_weight(weights.alloc.ref_time(), expected.alloc.ref_time());
+    check_syscall_weight(
+        weights.alloc_per_page.ref_time(),
+        expected.alloc_per_page.ref_time(),
+    );
```

Comment:
Could you please make the same using macro? so that this can be written:
```rust
...
check!(alloc);
check!(alloc_per_page);
check!(free);
...
```